### ISSUE TITLE
Add more rules

### ISSUE THREAD

**Issue by sublayerio** [armada-alliance/armada-alliance#156]
- Add Rule X: I declare that I've backed up my keys multiple times and I'm aware that there's no way in recovering them
- Add Rule X: Metadata hash should match registered hash
- Add Rule X: It's best practice to provide a link to a (limited) public grafana dashboard
- Add Rule X: It's best practice to have Armada Ping set up

**Comment by AstroWa3l** [comment]
Can we talk about this as a group before implementing any more rules?

**Comment by sublayerio** [comment]
Correct. The idea is to collect them here and decide on them later. Feel free to add to this list.

**Comment by AstroWa3l** [comment]
I do not agree with this new rules comment or issue.
